veracruz-client/src/cli.rs

```diff
@@ -11,27 +11,88 @@
 
 use structopt::StructOpt;
 use std::{
+    cell::RefCell,
+    cell::RefMut,
     fs,
     io,
     io::Read,
     io::Write,
+    num::NonZeroU32,
+    ops::Deref,
     path,
     process,
+    time::Duration,
+};
+use veracruz_client::{
+    VeracruzAdminClient,
+    VeracruzClient,
 };
-use veracruz_client::VeracruzClient;
 use veracruz_utils::policy::{
     error::PolicyError,
     parsers,
     policy::Policy,
 };
 
 
+/// A bit of extra parsing to allow omitting addr/port
+fn parse_bind_addr(s: &str) -> String {
```

Comment:
Should this be a `String` too to avoid an extra allocation if the caller already has a `String`?